src/styles/common/_navbar.scss

```diff
@@ -0,0 +1,40 @@
+.header__container {
+  background-color: $primary_color !important;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 1rem;
+}
+
+.header__icon {
+  color: $link_color !important;
```

Comment:
Too many `!important` s peppered around. Is this because we have to override carbon's styling? There should be a better way right?

Reply:
Yes, had to use this to override the carbon default colors, one other way I found was to use specific css selectors for classnames, so if we have a carbon component, we can use the classname we have assigned along with the classname carbon has to override the styles, something like .header__icon.bx--header--name. But I was not sure if this is a better way of achieving the same result?

Reply:
Yes, please use that. a higher specificity is always better than using !important. !important will make sure future code becomes much more problematic. In fact an easier way is to give your component a classname and use that as part of the selector so `.parent-component-name .your-class`, which adds the specificity. Do read up about css specificity rules so that it becomes easier to work with

Reply:
Okay will make sure to use css specificity to avoid !important usage